py/desitarget/sv3/sv3_cuts.py

```diff
@@ -1237,6 +1232,21 @@ def isBGS_colors(rfiberflux=None, gflux=None, rflux=None, zflux=None, w1flux=Non
         primary = np.ones_like(rflux, dtype='?')
     bgs = primary.copy()
     fmc = np.zeros_like(rflux, dtype='?')
+    
```

Comment:
In the previous version, we kept the `isBGS_lslga()` function (which should now be called `isBGS_sga()`, I guess) separate rather than embedding that part of the selection in `isBGS_colors()`. 

This was a deliberate choice because the mock simulations don't include a concept of `refcat`. Please return the code to its original structure, with a separate `isBGS_sga()` function.

Reply:
I see. I'll put it back.